dotnetv3/Glue/scenarios/Glue_Basics_Scenario/Glue_Basics/GlueBasics.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright Amazon.com, Inc.or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache - 2.0
+
+// snippet-start:[Glue.dotnetv3.GlueBasicsScenario.Main]
+// This example uses .NET Core 6 and the AWS SDK for .NET v3.7
```

Comment:
// This example uses .NET Core 6 and the AWS SDK for .NET (v3.7).

Reply:
changed